plugins/rapid7_insightvm/komand_rapid7_insightvm/triggers/scan_completion/trigger.py

```diff
@@ -174,49 +167,45 @@ def filter_results(params: dict, csv_row: dict, results: dict) -> Union[None, di
         source = params.get(Input.SOURCE, None)
         cvss_score = params.get(Input.CVSS_SCORE, None)
         severity = params.get(Input.SEVERITY, None)
+        category = params.get(Input.CATEGORY_NAME, "").lower()
 
         # We retrieve this separately because we use it as a unique identifier for
         # the filtering process
         asset_id = int(csv_row.get("asset_id", 0))
 
-        new_dict = {
+        asset_dict = {
             "asset_id": asset_id,
             "hostname": csv_row.get("host_name", ""),
             "ip_address": csv_row.get("ip_address", ""),
-            "vulnerability_info": [
-                {
-                    "vulnerability_id": csv_row.get("vulnerability_id", ""),
-                    "nexpose_id": csv_row.get("nexpose_id", ""),
-                    "cvss_v3_score": csv_row.get("cvss_v3_score", 0),
-                    "severity": csv_row.get("severity", ""),
-                    "solution_id": Util.strip_msft_id(csv_row.get("solution_id", "")),
-                    "solution_summary": csv_row.get("summary", ""),
-                }
-            ],
+        }
+
+        vulnerability_dict = {
+            "vulnerability_id": csv_row.get("vulnerability_id", ""),
```

Comment:
I know it's not in the ticket, but should `asset_id` not be added into this vulnerability? otherwise how do the vuln and assets get mapped together?